docs(seqera-ai): align Enterprise Co-Scientist docs for 26.1 (#1381)
seqeralabs/docs: platform-enterprise_docs/seqera-ai/authentication.md

When this environment variable is set, the CLI skips the OAuth login flow and uses the provided token directly.

### Connect to an Enterprise backend

Set the following environment variables before starting `seqera ai`:

| Variable | Purpose | Example value |
| --- | --- | --- |
| `SEQERA_AI_BACKEND_URL` | Co-Scientist backend endpoint used by the CLI | `https://ai-api.platform.example.com` |
| `SEQERA_AUTH_DOMAIN` | OIDC authority base URL. The CLI fetches OpenID configuration from this URL and opens the discovered authorization endpoint in your browser. | `https://platform.example.com/api` |

> | `SEQERA_AUTH_CLI_CLIENT_ID` | OAuth client ID for the Co-Scientist CLI | `seqera_ai_cli` |

Reviewer: ```suggestion
| `SEQERA_AUTH_CLI_CLIENT_ID` | OAuth client ID for the Seqera CLI | `seqera_ai_cli` |
```

## Organization management

> Co-Scientist CLI supports managing your organization selection for billing. Use the `seqera org` command to view and switch organizations.

Reviewer: ```suggestion
Seqera CLI supports managing your organization selection for billing. Use the `seqera org` command to view and switch organizations.
```

## Token refresh

> Co-Scientist CLI automatically refreshes your authentication token when needed. You are not required to log in again unless:

Reviewer: ```suggestion
Seqera CLI automatically refreshes your authentication token when needed. You are not required to log in again unless:
```

- You explicitly log out
- Your refresh token expires (typically after extended inactivity)
- Your Seqera Platform account permissions change

## Learn more

> - [Co-Scientist CLI](index.md): Co-Scientist CLI overview

Reviewer: ```suggestion
- [Seqera CLI](index.md): Seqera CLI overview
```
